lib/index.js

```diff
@@ -59,6 +59,11 @@ var ScrollContext = function (_Component) {
         value: function componentWillReceiveProps(nextProps) {
             this.checkScroll(nextProps.enable);
         }
+    }, {
+        key: 'componentWillUnmount',
```

Comment:
Why is this added now? Probably a previous code change was not published?

Reply:
Ah, I forgot to update `lib` in my PR https://github.com/varunpal/react-scroll-context/pull/2